src/it/scala/tech/cryptonomic/conseil/client/EndpointClient.scala

```diff
@@ -0,0 +1,169 @@
+package tech.cryptonomic.conseil.client
+
+import java.util.concurrent.Executors
+import scala.language.postfixOps
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration._
+import scala.sys.process._
+import scala.Console.{GREEN, RED, RESET}
+import org.http4s.client.blaze._
+import org.http4s.Header
+import org.http4s.headers._
+import org.http4s.MediaType
+import org.http4s.circe._
+import org.http4s.dsl.io._
+import org.http4s.client.dsl.io._
+import org.http4s.Uri
+import io.circe.{Json, parser}
+import cats.effect.IO
+import cats.syntax.apply._
+
+/** Currently can be used to test any conseil instance that loaded blocks levels 1 to 1000
+  * against predefined expectations on the responses
+  */
+object DataEndpointsClientProbe {
```

Comment:
Why not name this EndpointClient?

Reply:
The "Data" prefix is not necessarily correct, because other endpoints apart from `v2/data` are used for the test.

EndpointClient is too generic, it doesn't state what's his duty. It's too easy to take this for a client that gives you back data from the endpoint, like non-testing code.

EndpointsVerificationClient? EndpointsTester?

Open to suggestions